inference-engine/src/vpu/graph_transformer/src/stages/mtcnn.cpp

```diff
@@ -112,15 +112,10 @@ ie::CNNNetwork loadSubNetwork(
     // Load network
     //
 
+    // TODO: replace with ICore interface
```

Comment:
Could you please leave ticket number instead of `TODO`?

Reply:
fixed